tests/NuGet.Config

```diff
@@ -8,10 +8,14 @@
     <add key="automatic" value="True" />
   </packageRestore>
   <packageSources>
-    <add key="v2.nuget.org" value="https://nuget.org/api/v2" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="nugetbuild" value="https://dotnet.myget.org/F/nuget-build/api/v2/" />
   </packageSources>
   <activePackageSource>
     <add key="All" value="(Aggregate source)" />
   </activePackageSource>
+  <disabledPackageSources>
+    <clear />
+    <add key="https://www.nuget.org/api/v2/" value="true" />
```

Comment:
Why disable this?

Reply:
Copied from other NuGet.configs we have here and there.